setup.py

```diff
@@ -80,6 +80,7 @@ def is_requirement(line):
         'Development Status :: 4 - Beta',
         'Framework :: Django',
         'Framework :: Django :: 2.2',
+        'Framework :: Django :: 3.2',
```

Comment:
Here also, if we're testing 3.0 and 3.1 we should probably claim to support them.